io.github.mpobaschnig.Vaults.json

```diff
@@ -0,0 +1,69 @@
+{
+    "app-id" : "io.github.mpobaschnig.Vaults",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "vaults",
+    "finish-args" : [
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--talk-name=org.a11y.Bus",
+        "--talk-name=org.freedesktop.Flatpak",
+        "--filesystem=host"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [
+            "--share=network"
```

Comment:
this is not available with the builders.

you need to had the crate to the manifest. See https://github.com/flatpak/flatpak-builder-tools/tree/master/cargo for a tool that helps to do so.

Reply:
There's no need to as I believe the app is based on a template I made, you can run `meson _build && ninja -C _build dist` which will create a tarball you can use instead.

Reply:
I'm using a tarball now, thanks!